dowhy/causal_identifier/backdoor_identifier.py

```diff
@@ -1,32 +1,18 @@
-import copy
 import itertools
 import logging
+from enum import Enum
+from typing import List, Optional, Union
 
 import sympy as sp
 import sympy.stats as spstats
 
-import dowhy.utils.cli_helpers as cli
-from dowhy.causal_identifiers.efficient_backdoor import EfficientBackdoor
+from dowhy.causal_graph import CausalGraph
+from dowhy.causal_identifier.efficient_backdoor import EfficientBackdoor
+from dowhy.causal_identifier.identify_effect import CausalIdentifierEstimandType, IdentifiedEstimand
 from dowhy.utils.api import parse_state
 
 
-class CausalIdentifier:
-    """Class that implements different identification methods.
-
-    Currently supports backdoor and instrumental variable identification methods. The identification is based on the causal graph provided.
-
-    """
-
-    # Average total effect
-    NONPARAMETRIC_ATE = "nonparametric-ate"
-    # Natural direct effect
-    NONPARAMETRIC_NDE = "nonparametric-nde"
-    # Natural indirect effect
-    NONPARAMETRIC_NIE = "nonparametric-nie"
-    # Controlled direct effect
-    NONPARAMETRIC_CDE = "nonparametric-cde"
-
-    MAX_BACKDOOR_ITERATIONS = 100000
+class BackdoorAdjustmentMethod(Enum):
```

Comment:
Does this name of this enum make it sound like it's the actual method?  That is, like it's a callable method lambda instead of just an enum of the names of methods that we'll pass in as an argument?

Reply:
Ok.